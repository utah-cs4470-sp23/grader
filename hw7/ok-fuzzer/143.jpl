let a[b, c] = (array[a : (array[a : (- 901)] {argnum})[(argnum + (sum[a : 219] argnum))]{0}, b : (if (! (601 <= (sum[a : argnum, b : argnum] a))) then ((sum[a : (123 / argnum)] (- a)) + {args, 52, false}{1}) else (sum[a : (sum[a : argnum, b : (- 331), c : argnum] b)] argnum))] pow({sqrt(29.18)}{0}, (sum[c : (- a), d : (sum[c : b, d : argnum] (- argnum)), e : (sum[c : (sum[c : argnum, d : b, e : 233] a)] b)] (sum[f : (- a), g : e] tan(93.16)))))
let {d, e, f[g]} = (array[d : (array[d : argnum, e : 381] c)[(argnum % b), c], e : c, f : (624 % (argnum - argnum))] {true, {f, 291, args}, (if false then e else d)})[(if ((c == c) == (! true)) then (sum[d : argnum, e : (- c), f : (- argnum)] d) else b), 845, (sum[d : (array[d : 517, e : c, f : argnum] e)[b, (b - c), c], e : 494, f : argnum] args[(f - b)])]{1}
write image {(array[h : 117, i : b] {tan(15.53), to_float(b), cos(39.95), (if true then 64.64 else 21.33)})}{0} to "h.png"

let {h, i[j], {k}} = {{(true && (e > g)), (array[h : 735, i : e, j : e] args)[f[674], b, (635 + 273)], (if true then {d} else {e})}, (b <= c)}{0}
print "l"

print "l"

print "l"

let l[m, n, o] = (array[l : g, m : d] (array[n : m] (array[o : c, p : d, q : b] true)))[g, (if true then d else g)][450]
let p = 48.23
let {} = {m, (if (! true) then (array[q : b] {}) else (array[q : j] {}))}{1}[(if {(false && h), h, to_int(p)}{0} then g else o)]
write image (array[q : d, r : o] (if false then {sqrt((p / p)), (sum[s : (sum[s : m] g)] tan(52.91)), (sum[s : g, t : m, u : r] p), p} else {tan(p), (- (sum[s : c, t : m] p)), sin(atan(p)), {a, 25.48, d}{1}})) to "q.png"

show sqrt((sum[q : ({f, k}{1} % d), r : (- argnum), s : b] asin(sin(atan2(p, p)))))

