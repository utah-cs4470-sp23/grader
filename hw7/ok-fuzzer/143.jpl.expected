(LetCmd
 (ArgLValue (ArrayArgument a b c))
 (ArrayLoopExpr
  (ArrayType (FloatType) 2)
  a
  (TupleIndexExpr
   (IntType)
   (ArrayIndexExpr
    (TupleType (IntType))
    (ArrayLoopExpr
     (ArrayType (TupleType (IntType)) 1)
     a
     (UnopExpr (IntType) - (IntExpr (IntType) 901))
     (TupleLiteralExpr (TupleType (IntType)) (VarExpr (IntType) argnum)))
    (BinopExpr
     (IntType)
     (VarExpr (IntType) argnum)
     +
     (SumLoopExpr
      (IntType)
      a
      (IntExpr (IntType) 219)
      (VarExpr (IntType) argnum))))
   0)
  b
  (IfExpr
   (IntType)
   (UnopExpr
    (BoolType)
    !
    (BinopExpr
     (BoolType)
     (IntExpr (IntType) 601)
     <=
     (SumLoopExpr
      (IntType)
      a
      (VarExpr (IntType) argnum)
      b
      (VarExpr (IntType) argnum)
      (VarExpr (IntType) a))))
   (BinopExpr
    (IntType)
    (SumLoopExpr
     (IntType)
     a
     (BinopExpr (IntType) (IntExpr (IntType) 123) / (VarExpr (IntType) argnum))
     (UnopExpr (IntType) - (VarExpr (IntType) a)))
    +
    (TupleIndexExpr
     (IntType)
     (TupleLiteralExpr
      (TupleType (ArrayType (IntType) 1) (IntType) (BoolType))
      (VarExpr (ArrayType (IntType) 1) args)
      (IntExpr (IntType) 52)
      (FalseExpr (BoolType)))
     1))
   (SumLoopExpr
    (IntType)
    a
    (SumLoopExpr
     (IntType)
     a
     (VarExpr (IntType) argnum)
     b
     (UnopExpr (IntType) - (IntExpr (IntType) 331))
     c
     (VarExpr (IntType) argnum)
     (VarExpr (IntType) b))
    (VarExpr (IntType) argnum)))
  (CallExpr
   (FloatType)
   pow
   (TupleIndexExpr
    (FloatType)
    (TupleLiteralExpr
     (TupleType (FloatType))
     (CallExpr (FloatType) sqrt (FloatExpr (FloatType) 29)))
    0)
   (SumLoopExpr
    (FloatType)
    c
    (UnopExpr (IntType) - (VarExpr (IntType) a))
    d
    (SumLoopExpr
     (IntType)
     c
     (VarExpr (IntType) b)
     d
     (VarExpr (IntType) argnum)
     (UnopExpr (IntType) - (VarExpr (IntType) argnum)))
    e
    (SumLoopExpr
     (IntType)
     c
     (SumLoopExpr
      (IntType)
      c
      (VarExpr (IntType) argnum)
      d
      (VarExpr (IntType) b)
      e
      (IntExpr (IntType) 233)
      (VarExpr (IntType) a))
     (VarExpr (IntType) b))
    (SumLoopExpr
     (FloatType)
     f
     (UnopExpr (IntType) - (VarExpr (IntType) a))
     g
     (VarExpr (IntType) e)
     (CallExpr (FloatType) tan (FloatExpr (FloatType) 93)))))))
(LetCmd
 (TupleLValue
  (ArgLValue (VarArgument d))
  (ArgLValue (VarArgument e))
  (ArgLValue (ArrayArgument f g)))
 (TupleIndexExpr
  (TupleType (IntType) (IntType) (ArrayType (IntType) 1))
  (ArrayIndexExpr
   (TupleType
    (BoolType)
    (TupleType (IntType) (IntType) (ArrayType (IntType) 1))
    (IntType))
   (ArrayLoopExpr
    (ArrayType
     (TupleType
      (BoolType)
      (TupleType (IntType) (IntType) (ArrayType (IntType) 1))
      (IntType))
     3)
    d
    (ArrayIndexExpr
     (IntType)
     (ArrayLoopExpr
      (ArrayType (IntType) 2)
      d
      (VarExpr (IntType) argnum)
      e
      (IntExpr (IntType) 381)
      (VarExpr (IntType) c))
     (BinopExpr (IntType) (VarExpr (IntType) argnum) % (VarExpr (IntType) b))
     (VarExpr (IntType) c))
    e
    (VarExpr (IntType) c)
    f
    (BinopExpr
     (IntType)
     (IntExpr (IntType) 624)
     %
     (BinopExpr
      (IntType)
      (VarExpr (IntType) argnum)
      -
      (VarExpr (IntType) argnum)))
    (TupleLiteralExpr
     (TupleType
      (BoolType)
      (TupleType (IntType) (IntType) (ArrayType (IntType) 1))
      (IntType))
     (TrueExpr (BoolType))
     (TupleLiteralExpr
      (TupleType (IntType) (IntType) (ArrayType (IntType) 1))
      (VarExpr (IntType) f)
      (IntExpr (IntType) 291)
      (VarExpr (ArrayType (IntType) 1) args))
     (IfExpr
      (IntType)
      (FalseExpr (BoolType))
      (VarExpr (IntType) e)
      (VarExpr (IntType) d))))
   (IfExpr
    (IntType)
    (BinopExpr
     (BoolType)
     (BinopExpr (BoolType) (VarExpr (IntType) c) == (VarExpr (IntType) c))
     ==
     (UnopExpr (BoolType) ! (TrueExpr (BoolType))))
    (SumLoopExpr
     (IntType)
     d
     (VarExpr (IntType) argnum)
     e
     (UnopExpr (IntType) - (VarExpr (IntType) c))
     f
     (UnopExpr (IntType) - (VarExpr (IntType) argnum))
     (VarExpr (IntType) d))
    (VarExpr (IntType) b))
   (IntExpr (IntType) 845)
   (SumLoopExpr
    (IntType)
    d
    (ArrayIndexExpr
     (IntType)
     (ArrayLoopExpr
      (ArrayType (IntType) 3)
      d
      (IntExpr (IntType) 517)
      e
      (VarExpr (IntType) c)
      f
      (VarExpr (IntType) argnum)
      (VarExpr (IntType) e))
     (VarExpr (IntType) b)
     (BinopExpr (IntType) (VarExpr (IntType) b) - (VarExpr (IntType) c))
     (VarExpr (IntType) c))
    e
    (IntExpr (IntType) 494)
    f
    (VarExpr (IntType) argnum)
    (ArrayIndexExpr
     (IntType)
     (VarExpr (ArrayType (IntType) 1) args)
     (BinopExpr (IntType) (VarExpr (IntType) f) - (VarExpr (IntType) b)))))
  1))
(WriteCmd
 (TupleIndexExpr
  (ArrayType (TupleType (FloatType) (FloatType) (FloatType) (FloatType)) 2)
  (TupleLiteralExpr
   (TupleType
    (ArrayType (TupleType (FloatType) (FloatType) (FloatType) (FloatType)) 2))
   (ArrayLoopExpr
    (ArrayType (TupleType (FloatType) (FloatType) (FloatType) (FloatType)) 2)
    h
    (IntExpr (IntType) 117)
    i
    (VarExpr (IntType) b)
    (TupleLiteralExpr
     (TupleType (FloatType) (FloatType) (FloatType) (FloatType))
     (CallExpr (FloatType) tan (FloatExpr (FloatType) 15))
     (CallExpr (FloatType) to_float (VarExpr (IntType) b))
     (CallExpr (FloatType) cos (FloatExpr (FloatType) 39))
     (IfExpr
      (FloatType)
      (TrueExpr (BoolType))
      (FloatExpr (FloatType) 64)
      (FloatExpr (FloatType) 21)))))
  0)
 "h.png")
(LetCmd
 (TupleLValue
  (ArgLValue (VarArgument h))
  (ArgLValue (ArrayArgument i j))
  (TupleLValue (ArgLValue (VarArgument k))))
 (TupleIndexExpr
  (TupleType (BoolType) (ArrayType (IntType) 1) (TupleType (IntType)))
  (TupleLiteralExpr
   (TupleType
    (TupleType (BoolType) (ArrayType (IntType) 1) (TupleType (IntType)))
    (BoolType))
   (TupleLiteralExpr
    (TupleType (BoolType) (ArrayType (IntType) 1) (TupleType (IntType)))
    (BinopExpr
     (BoolType)
     (TrueExpr (BoolType))
     &&
     (BinopExpr (BoolType) (VarExpr (IntType) e) > (VarExpr (IntType) g)))
    (ArrayIndexExpr
     (ArrayType (IntType) 1)
     (ArrayLoopExpr
      (ArrayType (ArrayType (IntType) 1) 3)
      h
      (IntExpr (IntType) 735)
      i
      (VarExpr (IntType) e)
      j
      (VarExpr (IntType) e)
      (VarExpr (ArrayType (IntType) 1) args))
     (ArrayIndexExpr
      (IntType)
      (VarExpr (ArrayType (IntType) 1) f)
      (IntExpr (IntType) 674))
     (VarExpr (IntType) b)
     (BinopExpr (IntType) (IntExpr (IntType) 635) + (IntExpr (IntType) 273)))
    (IfExpr
     (TupleType (IntType))
     (TrueExpr (BoolType))
     (TupleLiteralExpr (TupleType (IntType)) (VarExpr (IntType) d))
     (TupleLiteralExpr (TupleType (IntType)) (VarExpr (IntType) e))))
   (BinopExpr (BoolType) (VarExpr (IntType) b) <= (VarExpr (IntType) c)))
  0))
(PrintCmd "l")
(PrintCmd "l")
(PrintCmd "l")
(LetCmd
 (ArgLValue (ArrayArgument l m n o))
 (ArrayIndexExpr
  (ArrayType (BoolType) 3)
  (ArrayIndexExpr
   (ArrayType (ArrayType (BoolType) 3) 1)
   (ArrayLoopExpr
    (ArrayType (ArrayType (ArrayType (BoolType) 3) 1) 2)
    l
    (VarExpr (IntType) g)
    m
    (VarExpr (IntType) d)
    (ArrayLoopExpr
     (ArrayType (ArrayType (BoolType) 3) 1)
     n
     (VarExpr (IntType) m)
     (ArrayLoopExpr
      (ArrayType (BoolType) 3)
      o
      (VarExpr (IntType) c)
      p
      (VarExpr (IntType) d)
      q
      (VarExpr (IntType) b)
      (TrueExpr (BoolType)))))
   (VarExpr (IntType) g)
   (IfExpr
    (IntType)
    (TrueExpr (BoolType))
    (VarExpr (IntType) d)
    (VarExpr (IntType) g)))
  (IntExpr (IntType) 450)))
(LetCmd (ArgLValue (VarArgument p)) (FloatExpr (FloatType) 48))
(LetCmd
 (TupleLValue)
 (ArrayIndexExpr
  (TupleType)
  (TupleIndexExpr
   (ArrayType (TupleType) 1)
   (TupleLiteralExpr
    (TupleType (IntType) (ArrayType (TupleType) 1))
    (VarExpr (IntType) m)
    (IfExpr
     (ArrayType (TupleType) 1)
     (UnopExpr (BoolType) ! (TrueExpr (BoolType)))
     (ArrayLoopExpr
      (ArrayType (TupleType) 1)
      q
      (VarExpr (IntType) b)
      (TupleLiteralExpr (TupleType)))
     (ArrayLoopExpr
      (ArrayType (TupleType) 1)
      q
      (VarExpr (IntType) j)
      (TupleLiteralExpr (TupleType)))))
   1)
  (IfExpr
   (IntType)
   (TupleIndexExpr
    (BoolType)
    (TupleLiteralExpr
     (TupleType (BoolType) (BoolType) (IntType))
     (BinopExpr (BoolType) (FalseExpr (BoolType)) && (VarExpr (BoolType) h))
     (VarExpr (BoolType) h)
     (CallExpr (IntType) to_int (VarExpr (FloatType) p)))
    0)
   (VarExpr (IntType) g)
   (VarExpr (IntType) o))))
(WriteCmd
 (ArrayLoopExpr
  (ArrayType (TupleType (FloatType) (FloatType) (FloatType) (FloatType)) 2)
  q
  (VarExpr (IntType) d)
  r
  (VarExpr (IntType) o)
  (IfExpr
   (TupleType (FloatType) (FloatType) (FloatType) (FloatType))
   (FalseExpr (BoolType))
   (TupleLiteralExpr
    (TupleType (FloatType) (FloatType) (FloatType) (FloatType))
    (CallExpr
     (FloatType)
     sqrt
     (BinopExpr (FloatType) (VarExpr (FloatType) p) / (VarExpr (FloatType) p)))
    (SumLoopExpr
     (FloatType)
     s
     (SumLoopExpr (IntType) s (VarExpr (IntType) m) (VarExpr (IntType) g))
     (CallExpr (FloatType) tan (FloatExpr (FloatType) 52)))
    (SumLoopExpr
     (FloatType)
     s
     (VarExpr (IntType) g)
     t
     (VarExpr (IntType) m)
     u
     (VarExpr (IntType) r)
     (VarExpr (FloatType) p))
    (VarExpr (FloatType) p))
   (TupleLiteralExpr
    (TupleType (FloatType) (FloatType) (FloatType) (FloatType))
    (CallExpr (FloatType) tan (VarExpr (FloatType) p))
    (UnopExpr
     (FloatType)
     -
     (SumLoopExpr
      (FloatType)
      s
      (VarExpr (IntType) c)
      t
      (VarExpr (IntType) m)
      (VarExpr (FloatType) p)))
    (CallExpr
     (FloatType)
     sin
     (CallExpr (FloatType) atan (VarExpr (FloatType) p)))
    (TupleIndexExpr
     (FloatType)
     (TupleLiteralExpr
      (TupleType (ArrayType (FloatType) 2) (FloatType) (IntType))
      (VarExpr (ArrayType (FloatType) 2) a)
      (FloatExpr (FloatType) 25)
      (VarExpr (IntType) d))
     1))))
 "q.png")
(ShowCmd
 (CallExpr
  (FloatType)
  sqrt
  (SumLoopExpr
   (FloatType)
   q
   (BinopExpr
    (IntType)
    (TupleIndexExpr
     (IntType)
     (TupleLiteralExpr
      (TupleType (ArrayType (IntType) 1) (IntType))
      (VarExpr (ArrayType (IntType) 1) f)
      (VarExpr (IntType) k))
     1)
    %
    (VarExpr (IntType) d))
   r
   (UnopExpr (IntType) - (VarExpr (IntType) argnum))
   s
   (VarExpr (IntType) b)
   (CallExpr
    (FloatType)
    asin
    (CallExpr
     (FloatType)
     sin
     (CallExpr
      (FloatType)
      atan2
      (VarExpr (FloatType) p)
      (VarExpr (FloatType) p)))))))
